<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgGame</class>
 <widget class="QWidget" name="wdgGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGraphicsView" name="view">
       <property name="backgroundBrush">
        <brush brushstyle="Dense4Pattern">
         <color alpha="255">
          <red>53</red>
          <green>137</green>
          <blue>17</blue>
         </color>
        </brush>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="text">
          <string>A grande:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="grpEnvidos">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="cmdPaso">
              <property name="text">
               <string>Paso</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmdQuiero">
              <property name="text">
               <string>Quiero</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="cmdEnvido">
                <property name="text">
                 <string>Envido</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtEnvido">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>2</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="cmdOrdago">
              <property name="text">
               <string>Órdago</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="grpMus">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="cmdHastaCompanero">
            <property name="text">
             <string>Hasta mi compañero</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cmdMus">
            <property name="text">
             <string>Mus</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cmdNoMus">
            <property name="text">
             <string>No hay mus</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="grpTengo">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="cmdTengo">
            <property name="text">
             <string>Tengo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cmdNoTengo">
            <property name="text">
             <string>No tengo</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
